Index: README.md
===================================================================
diff --git a/README.md b/README.md
deleted file mode 100644
--- a/README.md	(revision 831d5304ab24603ebe1bbe0ba3abe06ed5903de4)
+++ /dev/null	(revision 831d5304ab24603ebe1bbe0ba3abe06ed5903de4)
@@ -1,1 +0,0 @@
-# Inn
\ No newline at end of file
Index: Task_1/index.php
===================================================================
diff --git a/Task_1/index.php b/Task_1/index.php
deleted file mode 100644
--- a/Task_1/index.php	(revision 831d5304ab24603ebe1bbe0ba3abe06ed5903de4)
+++ /dev/null	(revision 831d5304ab24603ebe1bbe0ba3abe06ed5903de4)
@@ -1,8 +0,0 @@
-<?php
-
-function greatestNumber(int $inputNumber): string
-{
-    return $inputNumber > 30 ? "$inputNumber - greatest number"  : ($inputNumber > 20 ? "30" : ($inputNumber > 10 ? "20, 30" : "10, 20, 30"));
-}
-
-echo greatestNumber(15);
