Index: README.md
===================================================================
diff --git a/README.md b/README.md
deleted file mode 100644
--- a/README.md	(revision e177168875a20907c70b726e0f4fb3a26a284457)
+++ /dev/null	(revision e177168875a20907c70b726e0f4fb3a26a284457)
@@ -1,1 +0,0 @@
-# Inn
\ No newline at end of file
Index: Task_1/index.php
===================================================================
diff --git a/Task_1/index.php b/Task_1/index.php
deleted file mode 100644
--- a/Task_1/index.php	(revision e177168875a20907c70b726e0f4fb3a26a284457)
+++ /dev/null	(revision e177168875a20907c70b726e0f4fb3a26a284457)
@@ -1,8 +0,0 @@
-<?php
-
-function greatestNumber(int $a): string
-{
-    return $a > 30 ? "$a - greatest number"  : ($a > 20 ? "30" : ($a > 10 ? "20, 30" : "10, 20, 30"));
-}
-
-echo greatestNumber(15);
